CMakeLists.txt

```diff
@@ -223,10 +225,10 @@ else()
 	check_and_add_flag(VISIBILITY_INLINES_HIDDEN -fvisibility-inlines-hidden)
 	check_and_add_flag(VISIBILITY_HIDDEN -fvisibility=hidden)
 
-	check_c_compiler_flag(-fomit-frame-pointer FLAG_C_FOMIT_FRAME_POINTER)
-	if(FLAG_C_FOMIT_FRAME_POINTER)
-		add_compile_options($<$<CONFIG:Release>:-fomit-frame-pointer>)
-	endif()
+	check_and_add_flag(FOMIT_FRAME_POINTER -fomit-frame-pointer RELEASE)
+
+	add_compile_definitions(_DEBUG DEBUG)
```

Comment:
Yes, but for different compilers, it kinds of hurts me to have it alone naked, which is why I prefer to have them grouped by compiler, even if there's little repetition.